dev-packages/e2e-tests/test-applications/create-remix-app-express-vite-dev/tests/behaviour-server.test.ts

```diff
@@ -7,7 +7,13 @@ test('Sends two linked transactions (server & client) to Sentry', async ({ page
   // We use this to identify the transactions
   const testTag = uuid4();
 
-  // no server span here!
+  const httpServerTransactionPromise = waitForTransaction('create-remix-app-express-vite-dev', transactionEvent => {
```

Comment:
This is actually instrumented now, without us having to do anything - probably the express instrumentation, nice!